README.md

```diff
@@ -13,40 +13,72 @@
     <a href="https://github.com/featuretoggles/feature-toggles"><img alt="Github CI" src="https://github.com/featuretoggles/feature-toggles/workflows/Node%20CI/badge.svg"></a>
 </p>
 
-# Feature Toggling the best way (Build time)
+# Feature Toggles
 
-Currently, This is available as babel plugin.
+A _Babel plugin_ that helps remove _unfinished code_ from your JavaScript and Node Js application on _build time_. This is a configuration based feature toggling.
 
-> Are you struggling with the maintenance of an unfinished feature?
-> Are you tired of long-running branches?
-> Do you want to implement Continuous Delivery and Trunk Based Development?
+> ### The list of problems stopping you to productive, then you are at right place.
```

Comment:
Content improvement suggestion.

If you are looking for a solution to the below mentioned problems, then you are at the right place.